pkg/webview/view.proto

```diff
@@ -93,6 +104,7 @@ message Resource {
   LocalResourceInfo local_resource_info = 17;
   string runtime_status = 18;
   bool is_tiltfile = 19;
+  repeated BuildType target_types = 27;
```

Comment:
hmmmmm...build types and target types are different enums in our Go data model. Treating them as the same here is going to lead to a lot of confusion.

Can we make this more closely match the Go data model? maybe something like:

```
repeated TargetSpec specs;

message TargetSpec {
  name string
  type string
  has_live_update bool
}
```